### ISSUE TITLE
Next Week Plan (04-21-21) 

### ISSUE THREAD

**Issue by bagmk** [bagmk/Quantum_Machine_Learning_Express#9]
Plan for next week
- ~Analytically calculate circuit to investigate how ML is cutting line~ 
- study one particular circuit and data to replicate the same accuracy
- Adam optimizer: mini-batching 
- ~optimize learning rate~
- compare the result with blog website circuit 19

Meeting summary
- Adam optimizer: mini-batching
- pennylane gradient descent
- learning rate
- Discussion: Strength of Ph.D. 
- Discussion : Startup company to big company
- Leadership and what it means
